### ISSUE TITLE
GeoServer Travis Test Error

### ISSUE THREAD

**Issue by lucernae** [kartoza/geonode#501]
Currently GeoServer Travis Test will have an error due to build.geonode.org can't be accessed to download GeoServer war image.
Not sure what is the resolution will be.
But for this reason we will skip Travis check and let it red for that matrix until it resolved.
QGIS Server test travis check requirement should still apply.
Developers can merge if QGIS Server Travis check passed.

**Comment by lucernae** [comment]
Currently there are two issues:

We are allowed to use geo-solutions' hosted build server.

1. When we used geoserver image and data for 2.12.2 (like what it should be before). Some java package is missing

2. When we used geoserver image 2.13 and beyond. The api and the code have mismatch. GeoNode code expect to query layer resource by name, like `layer_name` but instead geoserver returned `workspace:layer_name`. Need further debug to understand this behaviour. 

Either way, it will just take a long time to investigate so we will leave this open for now. The geoserver test will still be error.
